javascript-algorithms-and-data-structures/pokemon-search-app/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Poke Search App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h2 id="hd">Pokemon Search app</h2>
+
+    <div class="container">
+      <div class="searchBar">
+        <input
+          type="text"
+          name="pokemon"
+          id="search-input"
+          placeholder="Enter Pokemon Name or ID"
+        />
+        <button id="search-btn">Search</button>
+      </div>
+
+      <div id="pokeOutput">
+        <div class="top-container">
+          <div class="image-container">
+            <div class="poke-name-num">
+              <span id="poke_name" class="poke-info">Name: </span>
+              <span id="poke_num" class="poke-info">#ID: </span>
+            </div>
+
+            <img
```

Comment:
Observation: Images are currently displayed on one side, resulting in extra space in the middle.
Suggestion: Instead of having the image on one side, you can opt to have it centered leaving equal space on both sides.